templates/rustdoc/topbar.html

```diff
@@ -6,7 +6,7 @@
 {%- include "header/topbar_begin.html" -%}{#
 extra whitespace unremovable, need to use html tags unaffacted by whitespace T_T
 #}<ul class="pure-menu-list">
-    {%- if 'krate' in __tera_context -%}
+    {%- if __tera_context.krate -%}
```

Comment:
```suggestion
    {%- if krate -%}
```

`__tera_context` I think is a string serialization of the context, you can just read the variable off the context and if it's `undefined` it will be false.

Reply:
Alternatively to be more explicit

```suggestion
    {%- if krate is undefined -%}
```

Reply:
I agree with @Nemo157 here, 

the correct thing to do to check if a thing exists in the context is just `{% if var_name %}...`. 

https://tera.netlify.app/docs/#if

Reply:
>     {%- if krate is undefined -%}

This didn't work :( I had to use `if krate`